packager/react-native-xcode.sh

```diff
@@ -80,10 +80,17 @@ if [[ "$CONFIGURATION" = "Debug" && "$PLATFORM_NAME" != "iphonesimulator" ]]; th
   echo "$IP.xip.io" > "$DEST/ip.txt"
 fi
 
+BUNDLE_FILE="$DEST/main.jsbundle"
+
 $NODE_BINARY "$REACT_NATIVE_DIR/local-cli/cli.js" bundle \
   --entry-file "$ENTRY_FILE" \
   --platform ios \
   --dev $DEV \
   --reset-cache \
-  --bundle-output "$DEST/main.jsbundle" \
+  --bundle-output $BUNDLE_FILE \
   --assets-dest "$DEST"
+
+if [[ ! $DEV && ! -f $BUNDLE_FILE ]]; then
+  echo "Error: File $BUNDLE_FILE does not exist."
```

Comment:
Let's provide a better error message. Something along the lines of "this must be a bug with React Native, please report it here <url>"